From 87a8fe90fe846fd90e8b3e3c76e27d43bbd17ef9 Mon Sep 17 00:00:00 2001
From: Sylwester Nawrocki <s.nawrocki@samsung.com>
Date: Fri, 21 Jun 2013 14:07:07 +0200
Subject: [PATCH 0293/1302] tizen_defconfig update

Enable MIPI CSI-2/DSIM PHY driver and geenric PHY framework.

Signed-off-by: Sylwester Nawrocki <s.nawrocki@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/configs/tizen_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/tizen_defconfig b/arch/arm/configs/tizen_defconfig
index 0ab7ac0..fd97c41 100644
--- a/arch/arm/configs/tizen_defconfig
+++ b/arch/arm/configs/tizen_defconfig
@@ -318,7 +318,6 @@ CONFIG_S3C_GPIO_TRACK=y
 CONFIG_S3C_ADC=y
 CONFIG_S5P_DEV_MFC=y
 CONFIG_S3C24XX_PWM=y
-CONFIG_S5P_SETUP_MIPIPHY=y
 CONFIG_SAMSUNG_DMADEV=y
 
 #
@@ -2923,6 +2922,8 @@ CONFIG_ARM_GIC=y
 CONFIG_GIC_NON_BANKED=y
 # CONFIG_IPACK_BUS is not set
 # CONFIG_RESET_CONTROLLER is not set
+CONFIG_GENERIC_PHY=y
+CONFIG_PHY_EXYNOS_MIPI_DSIM_CSIS=y
 
 #
 # File systems
-- 
1.8.3.2

